### PR TITLE
Add documentation about `tbot`'s diagnostics service

### PR DESCRIPTION
**Pull request by strideynet**
Adds information about how to enable the diagnostics service, and what endpoints it exposes.

### CODE REVIEW

**Review by rosstimothy — APPROVED** [review]
Thanks @strideynet!

File: docs/pages/machine-id/reference/diagnostics-service.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+You can configure the diagnostics service using the `--diag-addr` CLI parameter:
+
+```code
+$ tbot start -c my-config.yaml --diag-addr 127.0.0.1:1234
+```
+
+Or directly within the configuration file using `diag_addr`:
+
+```yaml
+diag_addr: 127.0.0.1:1234
```
Comment (stevenGravy) [inline docs/pages/machine-id/reference/diagnostics-service.mdx:28]: typically we give `3xxx` ports. The diagnostic one for `teleport` is `3000`.  Perhaps `3001`?

Comment (strideynet) [reply docs/pages/machine-id/reference/diagnostics-service.mdx:28]: Sounds reasonable - it's really up to the user here but makes sense to demonstrate a default that aligns with our default ports.

File: docs/pages/machine-id/reference/diagnostics-service.mdx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+## Endpoints
+
```
Comment (ptgott) [inline docs/pages/machine-id/reference/diagnostics-service.mdx:32]: I would add a quick intro sentence to this section here

**Review by ptgott — APPROVED** [review]

File: docs/pages/machine-id/reference/diagnostics-service.mdx (hunk 1 above)
Comment (strideynet) [reply docs/pages/machine-id/reference/diagnostics-service.mdx]: 4248ee6c34

File: docs/pages/machine-id/reference/diagnostics-service.mdx (hunk 2 above)
Comment (strideynet) [reply docs/pages/machine-id/reference/diagnostics-service.mdx:32]: b92e9adf47

**Review by ryanclark — APPROVED** [review]

**Pull request merged**